packages/gatsby/src/utils/adapter/manager.ts

```diff
@@ -467,6 +467,7 @@ function getRoutesManifest(): RoutesManifest {
         (redirect.isPermanent
           ? HTTP_STATUS_CODE.MOVED_PERMANENTLY_301
           : HTTP_STATUS_CODE.FOUND_302),
+      force: redirect.force,
```

Comment:
Following on previous comment - as `force` is example of a field that Gatsby itself doesn't use, I think that a bit more "robust" would be to pass all the fields in the redirect (apart of ones that are explicit).

Something along the lines of this:

```ts
// pick fields we explicitly handle, and pass remaining ones as "platform specific" fields
const { fromPath, toPath, statusCode, isPermanent, ignoreCase, redirectInBrowser, ...platformSpecificFields } = redirect
// note that `redirectInBrowser` field will be unused - if that is `true` there is redirect mechanism in browser runtime to handle the redirects (regardless of value of this field we always want all redirects handled by "the server")

addRoute({
  path: fromPath,
  // [...] rest of the fields we already have here adjusting for the fact we destructured `redirect` already
  // so we can omit `redirect.` from most things

  // pass any unknown platform specific fields so platform adapter can use them
  ...platformSpecificFields
}) 
```

I think this change would also help with the somewhat related `conditions` issue not being honored (that might require additional look in https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-adapter-netlify/src/route-handler.ts but there is code that was moved from previous plugins already so it might "just work" after actually passing `conditions` in)